### PR TITLE
add isMulticast() in IPAddress

### PR DESCRIPTION
**Pull request by nsong**
Reviewer: @rlane

### CODE REVIEW

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com//job/loxigen_pull_req/143/
Test PASSed.

**Comment by nsong** [comment]
@rlane kindly reminder

File: java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/IPv6Address.java [hunk 1]
```diff
@@ -115,6 +115,14 @@ public boolean isBroadcast() {
         return this.equals(NO_MASK);
     }
 
+    /**
+     * IPv4 multicast addresses are defined by the prefix ff00::/8
```
Comment (rlane) [inline java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/IPv6Address.java]: Nit: IPv6

File: java_gen/pre-written/src/test/java/org/projectfloodlight/openflow/types/IPAddressTest.java [hunk 2]
```diff
@@ -130,4 +130,17 @@ public void testContainsException() {
         }
     }
 
+    @Test
+    public void testMulticastIp() {
+        IPAddress<?> ip0 = IPAddress.of("240.2.3.4");
+        IPAddress<?> ip1 = IPAddress.of("224.0.0.0");
```
Comment (rlane) [inline java_gen/pre-written/src/test/java/org/projectfloodlight/openflow/types/IPAddressTest.java]: Should have a test with some bits set after the multicast prefix.

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com//job/loxigen_pull_req/145/
Test PASSed.

**Comment by nsong** [comment]
SELF-ACK

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  Log file at http://vss1/abat/2015.05.15.0248-m.loxigen.master/abat.log

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
Log file at http://vss1/abat/2015.05.15.0248-m.loxigen.master/abat.log
/cc

**Pull request merged**